src/Listener/AddFlagsApiAttributes.php

```diff
@@ -44,6 +44,7 @@ public function handle(Serializing $event)
             }
 
             $event->attributes['guidelinesUrl'] = $this->settings->get('flarum-flags.guidelines_url');
+            $event->attributes['canFlagOwn'] = $this->settings->get('flarum-flags.can_flag_own');
```

Comment:
It would be easier to review / merge this quickly, if separate changes were in separate pull requests. :wink: 

@jordanjay29 Do you remember whether we made a decision about this? Is this flag useful enough to include in our bundled extension?

Reply:
@franzliedke I don't think we made a decision about it, no. It would be incredibly useful for Flarum.org as well, and I don't think any objections were raised to including it.

Reply:
Okay, let's keep it then.